UnitsNet.Tests/CustomCode/VolumeTests.cs

```diff
@@ -48,14 +48,24 @@ public class VolumeTests : VolumeTestsBase
         protected override double CubicYardsInOneCubicMeter => 1.30795062;
 
         protected override double DecilitersInOneCubicMeter => 1E4;
+        protected override double HectocubicFeetInOneCubicMeter => 3.531472e-1;
+        protected override double HectocubicMetersInOneCubicMeter => 0.01;
 
         protected override double HectolitersInOneCubicMeter => 1E1;
 
         protected override double ImperialGallonsInOneCubicMeter => 219.96924;
 
         protected override double ImperialOuncesInOneCubicMeter => 35195.07972;
 
+        protected override double KilocubicFeetInOneCubicMeter => 3.531472e-2;
+        protected override double KilocubicMetersInOneCubicMeter => 0.001;
+        protected override double KiloimperialGallonsInOneCubicMeter => 2.1996924e-1;
+        protected override double KilousGallonsInOneCubicMeter => 2.6417217e-1;
+
         protected override double LitersInOneCubicMeter => 1E3;
+        protected override double MegacubicFeetInOneCubicMeter => 3.531472e-5;
```

Comment:
And the megacubic ones too.

Reply:
The same here, they called it **Kcm (Thousand Cubic meters)**